hydromt/data_adapter/dataframe.py

```diff
@@ -136,14 +136,15 @@ def __init__(
 
     def to_file(
         self,
-        data_root,
-        data_name,
-        driver=None,
-        variables=None,
-        time_tuple=None,
-        logger=logger,
+        data_root: StrPath,
+        data_name: str,
+        driver: Optional[str] = None,
+        variables: Optional[Variables] = None,
+        time_tuple: Optional[TimeRange] = None,
+        handle_nodata: NoDataStrategy = NoDataStrategy.IGNORE,
```

Comment:
I think the default can still be RAISE no? Only for data_catalog.export_data I prefer to have IGNORE